scripts/globals/mobskills/Bubble_Shower.lua

```diff
@@ -10,6 +10,23 @@ require("scripts/globals/monstertpmoves");
 ---------------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
+	mob = mob:getID();
+	if (mob == 17129519) then

```

Comment:
@Musashi- just wanted to let you know that checking `mob:getPool()` will soon be an option as well, in the event checking the skill list won't do the job.